VoodooI2CHID/VoodooI2CMultitouchHIDEventDriver.cpp

```diff
@@ -750,7 +743,7 @@ bool VoodooI2CMultitouchHIDEventDriver::start(IOService* provider) {
     if (!super::start(provider))
         return false;
     
-    work_loop = this->getWorkLoop();
```

Comment:
Why removing `this` everywhere?

Reply:
It seems redundant. I can revert this commit if you want.